scripts/local-deploy.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "forge-std/Script.sol";
+import "../src/CosmoShips.sol";
+import "../src/GameLeague.sol";
+import "../src/IAttributeVerifier.sol";
+import "../test/fixtures/mockVerifier.sol";
+import "../test/fixtures/mockRandomGenerator.sol";
+
+contract MyScript is Script {
+    function run() external {
```

Comment:
`function run(string memory _root, uint256 _mintPrice) external {`
or
`function run(string memory _root, uint256 _mintPrice, address _defaultAdmin) external {`

Reply:
Adding those gives warnings.
I took the example from [here](https://book.getfoundry.sh/tutorials/solidity-scripting)

Reply:
resolved